cgi-bin/LJ/Widget.pm

```diff
@@ -146,6 +146,9 @@ sub render {
 
         #        $class->handle_error;
     }
+    else {
+        return;
```

Comment:
I'm curious as to why you added a null return here. Shouldn't you be returning an empty string if you don't expect to see any results, like when `$class->should_render` is false above?

Reply:
Hmmm empty string might work there, yeah - I added it because otherwise LJ::Widget passes back a bunch of wrapper HTML with no inner content, which makes Panel wrap it in more wrapper HTML at https://github.com/dreamwidth/dreamwidth/pull/3183/files#diff-73f66f3da82438b1bba42bd02af26110ad3fe2556794b7bb2af891bb7db2928dR113 , and then you get weird display because some of the CSS works on the wrapper classes.